web/packages/shared/components/DemoTerminal/helpers.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { DefaultTheme } from 'styled-components';
+
+import { emphasize } from 'design/theme/utils/colorManipulator';
+
+/** Returns color of the top bar of DemoTerminal. */
+export const topBarColor = (theme: DefaultTheme): string =>
+  emphasize(theme.colors.levels.deep, 0.2);
```

Comment:
This will be used in a later commit to stack two DemoTerminals on top of each other without creating a visible gap between them (since a DemoTerminal has border radius).